nabcommon/nabservice.py

```diff
@@ -147,6 +147,20 @@ async def stop_service_loop(self):
         """
         return None
 
+    @staticmethod
+    def get_nabd_host():
+        """
+        Returns the hostname to use to connect to nabd.
+        Typically that will be localhost / 127.0.0.1 however
+        when working with the Docker-based development environment
+        we connect to a dummy nabd daemon running in a separate
+        container under a specific hostname "nabdevd"
+        """
+        if os.getenv("PYNAB_DEVELOPMENT") is not None:
```

Comment:
Can we:
- have a specific environment variable containing the hostname (NABD_HOST)
- refactor this with:
```
class NabService:
...
    HOST = os.getenv("NABD_HOST", "127.0.0.1")
```

- symmetrically add NABD_PORT_NUMBER for PORT_NUMBER

Reply:
We certainly can, I did what I could with the little Python knowledge I have. I pushed a new commit for this.

Note that `PYNAB_DEVELOPMENT` is already used [there](https://github.com/nabaztag2018/pynab/blob/master/nabweb/urls.py#L76) from a previous PR. Let me know if I should handle it differently.